api/app/signals/apps/feedback/models.py

```diff
@@ -18,11 +18,23 @@ def generate_token():
     return uuid.UUID(hex=secrets.token_hex(16))
 
 
+class StandardAnswerTopic(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    text = models.TextField(max_length=1000)
```

Comment:
Agreed, if this is a description of the topic please name it so. 
However is this really needed? It is never exposed or used anywhere so perhaps it can be deleted.

Reply:
Yes the name can also be considered as description, as it is primarly used for the admin panel to easily distinguish which topic name is connected to a StanderdAnswer instead of only seeing an id or huge text